Задание_3/Задание_3/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Задание_3
+{
+    internal class Program
```

Comment:
Зачем `internal`? Вы хотите, чтобы класс `Program` был виден другим классам (которых нет) на уровне сборки?
Лучше просто `private` (или без модификатора, что то же самое).
```suggestion
    class Program
```